syntax-highlighting-code-block.php

```diff
@@ -145,12 +154,22 @@ function render_block( $attributes, $content ) {
 		$attributes['language'] = '';
 	}
 
+	if ( ! isset( $attributes['showLines'] ) ) {
+		$attributes['showLines'] = false;
+	}
+
 	// Enqueue the style now that we know it will be needed.
-	wp_enqueue_style( FRONTEND_STYLE_HANDLE );
+	wp_enqueue_style( FRONTEND_HIGHLIGHT_STYLE_HANDLE );
+	wp_enqueue_style( FRONTEND_PLUGIN_STYLE_HANDLE );
```

Comment:
As opposed to registering another external style to block rendering, let's perhaps instead use `wp_add_inline_style()`. The file can still be used, but the contents can then be obtained via `file_get_contents()`.

Reply:
Would using `wp_add_inline_style()` still need to call `wp_register_style()` or would it use `FRONTEND_HIGHLIGHT_STYLE_HANDLE`?

Reply:
Correct, it would use `FRONTEND_HIGHLIGHT_STYLE_HANDLE`. It would output the inline CSS when it prints the external link.